docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+---
+title: "Project property settings in Visual Studio"
+description: "Describes the Visual Studio project properties available to a .NET MAUI app. The properties and settings are related to building the app, configuring debug options, and other settings related to building the application for Windows, Android, and iOS."
+ms.date: 07/11/2022
+---
+
+# Project configuration in .NET MAUI
+
+.NET MAUI uses a [single-project system](../fundamentals/single-project.md) to manage most of your multi-platform app. This is different from .NET MAUI's predecessor, Xamarin, which used multiple projects, one for each platform. Project configuration in .NET MAUI is similar to other projects in Visual Studio, right-click on the project in the **Solution Explorer**, and select **Properties**.
+
+## Application
+
+The **Application** section describes some settings related to which platforms your app targets, as well as the output file and default namespace.
+
+- **General**
+
+  Describes some basic settings about your app.
+
+  | Setting | Default value | Description |
+  | - | - | - |
+  | Assembly name | `$(MSBuildProjectName)` | Specifies the name of the output file that will hold the assembly manifest. |
+  | Default namespace | Varies. | Specifies the base namespace for files added to your project. This generally defaults to the name of your project or a value you specified when you created the project. |
+
+- **iOS Targets**
+
+  If you're going to target iOS and macOS (using Mac Catalyst), these settings describe the target iOS version.
+
+  | Setting | Default value | Description |
+  | - | - | - |
+  | Target the iOS platform | Checked | Specifies that this project will target the iOS platform. |
+  | Target iOS Framework | `net6.0-ios` | The [Target Framework Moniker][tfm] used to target iOS.
+  | Minimum Target iOS Framework | `14.2` | The minimum version of iOS your app targets.
```

Comment:
No action needed here but I'm already sighing at the thought of having to keep these default values updated, as they change over time.